### PR TITLE
STOCK-3-InsertionStockMarketInformation

### PR DESCRIPTION
**Pull request by MahendraChinthala**
Please Check

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/stockAddPage.jsp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+<body>
+<div class="login">
+    <div class="login-screen">
+        <div class="app-title">
+            <h1>Stock Market Company Registration</h1>
+        </div>
+        <c:url value="/saveCompany" var="save"></c:url>
+
+        <div class="login-form">
+            <form name='loginForm' action="${save}" method='POST'>
```
Comment (pallavidongre) [inline src/main/webapp/WEB-INF/views/stockAddPage.jsp]: Use spring form tag library

File: src/main/java/com/nendrasys/controller/StockInformationController.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ModelAndView model = new ModelAndView();
+        model.setViewName("stockAddPage");
+        return model;
+    }
+
+    @RequestMapping(value = "/saveCompany", method = RequestMethod.POST)
+    public String saveCompany(StockDetailsModel stockDetailsModel) {
+        logger.info("Inside saveCompany method at StockInformationController class");
+        stockDetailsService.addStockToMarketList(stockDetailsModel);
+        return "redirect:/getAllStockMarketInformation";
```
Comment (pallavidongre) [inline src/main/java/com/nendrasys/controller/StockInformationController.java]: add success or failure message and display that message on appropriate jsp

File: src/main/webapp/WEB-INF/views/stockAddPage.jsp (hunk 1 above)
Comment (MahendraChinthala) [reply src/main/webapp/WEB-INF/views/stockAddPage.jsp]: Okay mam

File: src/main/java/com/nendrasys/controller/StockInformationController.java (hunk 2 above)
Comment (MahendraChinthala) [reply src/main/java/com/nendrasys/controller/StockInformationController.java]: Okay i will give Success jsp page mam

**Comment by pallavidongre** [comment]
resolve the conflicts

**Comment by MahendraChinthala** [comment]
I Resolve Conflicts Please Check one's mam

**Pull request merged**